src/mechanics/tools/CellStorage.cpp

```diff
@@ -0,0 +1,16 @@
+#include "mechanics/tools/CellStorage.h"
+#include "mechanics/cell/Cell.h"
+
+using namespace NuTo;
+
+Group<CellInterface> CellStorage::AddCells(Group<ElementCollectionFem> elements,
+                                           const IntegrationTypeBase& integrationType, int cellStartId)
```

Comment:
how do we ensure that this id is then unique?

Reply:
It is the users task to match the cell numbering with his history data numbering. If we want to make that a library task, we have to impose some common interface on all history data classes (to size/resize them). This may be too much of a restriction.